react-search-component/my-app/src/RenderedList.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+export default function List({ items }: Props) {
```

Comment:
You should name the component to match the file name. Doesn't matter which one you change, just make them match!